apis/scheduling/v1alpha1/reservation_types.go

```diff
@@ -123,9 +149,8 @@ type ReservationCondition struct {
 // Reservation is the Schema for the reservation API
 type Reservation struct {
 	metav1.TypeMeta `json:",inline"`
-	// A Reservation object is non-namespaced.
-	// It can reserve resources for pods of any namespace. Any affinity/anti-affinity of reservation scheduling can be
-	// specified in the pod template.
+	// A Reservation object is namespaced. But it can reserve resources for pods of any namespace. Any
```

Comment:
it's better to move the comment to the head of Reservation definition.

Reply:
done